config/fam_metadata_config.yaml

```diff
@@ -3,9 +3,8 @@
 # Metadata manager: default is radixtree. Currently radixtree is the only supported KVS.
 metadata_manager: radixtree
  
-# Path where metadata info is stored. By default, it will be placed in /dev/shm/<metadata_manager>. 
-# Example: If metadata manager is radixtree, then metadata info would be placed under /dev/shm/radixtree folder.
-metadata_path: /dev/shm/
+# Path where metadata info is stored. By default, it will be placed in /dev/shm/fam_metadata. 
```

Comment:
You can modify it to "By default, it will be placed in /dev/shm/<username>". If we leave this field empty that is where it is created.